src/Config.ts

```diff
@@ -73,13 +73,17 @@ export class Config {
 
     public readonly access: IConfigAccessControl = { };
 
-    public getRoomRule(roomIdOrAlias?: string) {
-        const aliasRule = this.roomRules.find((r) => r.room === roomIdOrAlias);
-        if (aliasRule && aliasRule.action === "deny") {
+    public getRoomRule(roomIdOrAlias: string) {
```

Comment:
This is a "first match wins" algorithm. I don't think this is expected by users when supporting regexps.

Without an annotation in the config sample (and the docs) I would expect exceptions like these to work:
```
/discord/ deny
"discord-jaller" allow
```

I think that would be `.findLast()` (<-- not a JS function). But we can also have the annotation saying that the first match wins.